### DISCUSSION TITLE
Is there a way to use IOS/Android system font?

### DISCUSSION THREAD

**Discussion by ziranjameshe** [tamagui/tamagui#2135] [Q&A]
At the moment, on iOS, it seems inter is the default font. Is there a way I can just use the default iOS font?

**Comment by likeSo** [comment]
I have this question too... Maybe this doc can help? But I don't know how to use
https://tamagui.dev/docs/core/configuration#font-tokens

**Comment by natew** [comment]
Just use `createFont` to make a font and use the family `System`

**Reply by natew** [reply]
If you're using `tamagui` then make that font for `heading` and `body`
